site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
-
-1.  Sign in to your [Chrome Web Store dashboard page][1].
-2.  If you don't see a drop-down at the top of dashboard page, go to step 3.
-3.  If you do see a drop-down, you've already been added to group publishing by another developer.
-    From the drop-down, select your own account instead of the group publisher account.
-4.  At the bottom of the dashboard page, you'll see one of two links:
-    - **Google Group publishing: Choose existing group or create new group** This link shows up if
-      you're the owner of one or more Google Groups, and those groups haven't already been linked to
-      a publisher account.
-    - **Google Group publishing: Create new group** You'll see this link if you aren't an owner of
-      any Google Groups, or if all the groups that you are an owner of have already been linked to
-      other publisher accounts.
-5.  You can do one of the following:
-    - Create a new Google group and link to this new group: 1. Click **Create new group**. You'll
-      see the Google Groups page in a new tab. 2. Create your new group. 3. Reload the Chrome Web
-      Store dashboard page in your browser, then follow the steps below under "Link to an existing
-      Google Group."
-    - Link to an existing Google group: 1. Click **Choose existing group**. 2. Select a group from
-      the drop-down. (We recommend linking to a group that you're using only for group publishing of
-      your items.) 3. Click **OK** in the confirmation box. 4. Click **Save Changes** at the bottom
-      of dashboard page.
-
-After you complete the steps above, a new Group Publisher account is created. The Google Group you
-selected or created is linked to this new publisher account, and the group email is the new
-publisher account's email.
-
-## After you set up group publishing
-
-Your dashboard page will show the new Group Publisher account and the linked Google Group. You can
-[add or remove developers][2]. **All members of the linked group can act on behalf of the new
-publisher account**. For example, they can edit items, publish items, edit the publisher's display
-name, etc.
+* A Chrome Web Store developer account can only create one group publisher,
+  **ever**. You cannot replenish this quota, even if you delete the group. 
+* Once the group publisher account is created, the Google Group you select 
+  remains linked to that account. Only the group owner or manager (or the last
+  remaining group member) can delete the group publisher, unlinking the Google 
+  Group from the Chrome Web Store.
+
+    <p class="warning">
+    Deleting the group publisher and the Google Group does not restore your
+  lifetime quota of one group publisher activation.
+    </p>
+
+* You can be a *member* of any number of group publishers.
+
+* You cannot change which group is linked to the Group Publisher account.
+* You still retain your personal publisherx account and can publish from either your group or personal account.
```

Comment:
You called it individual publisher before, and here it's called personal account. Shall we stick with only one name? personal publisher vs individual publisher.

Reply:
Done